src/Umbraco.Tests.Integration/Testing/UmbracoIntegrationTest.cs

```diff
@@ -257,14 +258,15 @@ public virtual void ConfigureServices(IServiceCollection services)
 
             // Add it!
             services.AddUmbracoConfiguration(Configuration);
-
-            services.AddUmbracoCore(
+            var builder = new UmbracoBuilder(services, Configuration);
```

Comment:
When we refactor the tests. This is screaming for https://github.com/umbraco/Umbraco-CMS/issues/8654